src/json/config.schema.json

```diff
@@ -142,6 +142,18 @@
             "title": "Checksum command",
             "examples": ["sha256sum FILENAME"]
           },
+          "checksumActionCommand": {
+            "$id": "#/properties/platforms/items/properties/checksumActionCommand",
+            "$ref": "#/definitions/command",
+            "title": "Checksum auto command",
+            "examples": ["sha256sum FILENAME"]
+          },
+          "checksumActionCommandResult": {
+            "$id": "#/properties/platforms/items/properties/checksumActionResultCommand",
+            "type": "string",
```

Comment:
the `#id` should be `#/properties/platforms/items/properties/checksumAutoResultCommand`